index.js

```diff
@@ -109,7 +109,7 @@ function shortenURL(href, currentUrl = 'https://github.com') {
 	const [, downloadTag, downloadFilename] = repoPath.match(releaseDownloadRegex) || [];
 	const [, label] = repoPath.match(labelRegex) || [];
 	const [, compare] = repoPath.match(compareRegex) || [];
-	const [, pull, pullPage, pullPartialStart, pullPartialEnd] = repoPath.match(pullRegex) || [];
+	const {pull, pullPage, pullPartialStart, pullPartialEnd} = (repoPath.match(pullRegex) || {groups: {}}).groups;
```

Comment:
I'm not a super fan of this but alas

Reply:
> I'm not a super fan of this

Me neither!
I didn't use optional chaining because I thought it wasn't yet supported in JS, but if we only target latest browsers it's possible: https://caniuse.com/?search=optional%20chaining

Reply:
This is indeed better:

```suggestion
	const {pull, pullPage, pullPartialStart, pullPartialEnd} = repoPath.match(pullRegex)?.groups ?? {}
```

But it would become a breaking change… which we can do since we're basically the only users. Let's see if there's anything else that would be a breaking change.

Last breaking release for reference: https://github.com/refined-github/shorten-repo-url/releases/tag/v2.0.0

Reply:
http://node.green

Reply:
> Let's see if there's anything else that would be a breaking change

If we limit support to Node 14+, might as well use nullish coalescing everywhere instead of `||`.